maps/models.py

```diff
@@ -272,6 +272,25 @@ class Map(models.Model):
         null=True, blank=True
     )
 
+    # Decision Making/Audience targets group
+    # Decision making/audience targets
+    has_explicit_indication_of_target_audience = models.BooleanField(
+        default=False
+    )
+    explicit_target_audience_text_explanation = models.TextField(
+        blank=True, null=True,
+    )
+    has_potential_target_audience = models.BooleanField(default=False)
```

Comment:
Appears as a nested conditional which doesn't collapse in the Form. Should this be in a different block?